src/block/tile/BrewingStand.php

```diff
@@ -83,6 +91,14 @@ protected function writeSaveData(CompoundTag $nbt) : void{
 		$nbt->setShort(self::TAG_REMAINING_FUEL_TIME_PE, $this->remainingFuelTime);
 	}
 
+	protected function addAdditionalSpawnData(CompoundTag $nbt) : void{
+		$this->addNameSpawnData($nbt);
+
+		$nbt->setShort(self::TAG_BREW_TIME_PE, $this->brewTime);
+		$nbt->setShort(self::TAG_MAX_FUEL_TIME, $this->maxFuelTime);
+		$nbt->setShort(self::TAG_REMAINING_FUEL_TIME_PE, $this->remainingFuelTime);
+	}
+
```

Comment:
Are these really necessary?

Reply:
Brew time is not really necessary, but others are in order to inform client about state of the fuel when update is not running.